src/components/BugScreenshotDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import { IpcClient } from "@/ipc/ipc_client";
+import { Dialog, DialogTitle } from "@radix-ui/react-dialog";
+import { DialogContent, DialogHeader } from "./ui/dialog";
+import { Button } from "./ui/button";
+import { BugIcon, Camera } from "lucide-react";
+import { useState } from "react";
+import { ScreenshotSuccessDialog } from "./ScreenshotSuccessDialog";
+
+interface BugScreenshotDialogProps {
+  isOpen: boolean;
+  onClose: () => void;
+  handleReportBug: () => Promise<void>;
+  isLoading: boolean;
+}
+export function BugScreenshotDialog({
+  isOpen,
+  onClose,
+  handleReportBug,
+  isLoading,
+}: BugScreenshotDialogProps) {
+  const [isScreenshotSuccessOpen, setIsScreenshotSuccessOpen] = useState(false);
+  const handleReportBugWithScreenshot = async () => {
+    onClose();
+    setTimeout(async () => {
+      try {
+        await IpcClient.getInstance().takeScreenshot();
+        setIsScreenshotSuccessOpen(true);
+      } catch (error) {
+        console.error("Couldn't take a screenshot", error);
```

Comment:
if we can't take a screenshot, i'd just show an error text in the dialog and then the user can just manually click the report bug button (I don't think we need to call handleReportBug here)